src/main/java/com/courtcanva/ccfranchise/repositories/OrderRepository.java

```diff
@@ -0,0 +1,13 @@
+package com.courtcanva.ccfranchise.repositories;
+
+import com.courtcanva.ccfranchise.constants.OrderStatus;
+import com.courtcanva.ccfranchise.models.Order;
+import org.springframework.data.jpa.repository.JpaRepository;
+import java.util.List;
+
+public interface OrderRepository extends JpaRepository<Order, Long> {
+
+    List<Order> findFirst10ByFranchiseeId(Long franchiseeId);
```

Comment:
I think it's better to use Optional when you find something

Reply:
this search is to return a list, so List is fine.  But still, this is using First10, doesn't really match where It is used.

Reply:
get First10 is the requirment from BA.  I added First10 in method names in Controller and Service to match `findFirst10ByFranchiseeId`

Reply:
not useful at current stage, deleted.